client/src/components/createContestComponents/CreateContest.js

```diff
@@ -0,0 +1,60 @@
+import React, { useContext } from 'react';
+import axios from 'axios';
+import { Typography, Box, Button } from '@material-ui/core';
+import { ContestContext } from './ContestContext';
+import useStyles from './CreateContestStyles';
+
+export const CreateContest = () => {
+  const classes = useStyles();
+  const contest = useContext(ContestContext);
+
+  const title = contest.title;
+  const description = contest.description;
+  const prize = contest.prize;
+  const deadline = contest.selectedDate;
+  const contestPics = contest.pics;
```

Comment:
```suggestion
  const {title, description, prize, selectedDate: deadline, pics: contestPics} = contest;
```

can take advantage of es6 deconstruction methods to really shorten the num lines of code here :)

Reply:
This has been updated. Good call. Thanks.